### DISCUSSION TITLE
How to spot unused CSS variables

### DISCUSSION THREAD

**Discussion by ghost** [twbs/bootstrap#38274] [Q&A] [answered]
Hi,
For people like me who are not that good in the front-end, I have a web-app in the other hand which relies mostly on Bootstrap.
I'm not looking for performance gains as I've already gone through this: https://getbootstrap.com/docs/5.0/customize/optimize/

I have the impression, (maybe wrong) that the ones who did the front end job before me used a lot of variables in a way, probably to just make it work somehow.
Also, probably some variables are no longer used in V5 ? who knows because it has been developed on Bootstrap V4 and I updated to V5 because the new dark mode is wahoo!

So are there any tools (preferably) or a way to spot theses classes  ?

Thanks a lot .

**Comment by patkon** [comment, marked as answer]
Maybe this is a solution for you:
https://css-tricks.com/detect-unused-classes-in-html/

**Reply by ghost** [reply]
thanks a lot @patkon 
it looks like the exact solution for me.
